src/bonuses.cpp

```diff
@@ -34,7 +34,7 @@ static const std::map<std::string, scaling_stat> scaling_stat_map = {{
         std::make_pair( "str", STAT_STR ),
         std::make_pair( "dex", STAT_DEX ),
         std::make_pair( "int", STAT_INT ),
-        std::make_pair( "per", STAT_PER ),
+        std::make_pair( "per", STAT_PER )
```

Comment:
You should leave the comma here. It's valid. And if somebody has to add another line, they don't need to change the previous lne just for adding the comma to it.